src/datagrid/components/Table/CellRenderer.tsx

```diff
@@ -151,6 +170,42 @@ const _DataCell = (
         }));
     }, [onFocus, setFocusCellPluginStore]);
 
+    const leftElement = useMemo(
+        () =>
+            left ? (
```

Comment:
wrap the entire thing in fragment or wrap nothing at all.

Reply:
same with right

Reply:
resolved @shashank1010